loom/script/runtime/lsLuaState.cpp

```diff
@@ -87,6 +87,7 @@ void LSLuaState::open()
     assert(!L);
 
     L = lua_newstate(lsLuaAlloc, NULL);
+    //L = luaL_newstate();
```

Comment:
That one is the default-allocation version of `lua_newstate`. I think `lua_open` was deprecated and is not available anymore. If was defined as `luaL_newstate` anyway though.